packages/experimental/Button/MIGRATION.md

```diff
@@ -0,0 +1,52 @@
+# Button Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Button
+
+### Component renames
+
+Common buttons now all map to `Button`:
+
+| v0 `Button`                             | v1 `Button`                                          |
+| --------------------------------------- | ---------------------------------------------------- |
+| `<Button text="Hello, world" />`        | `<Button>Hello, world</Button>`                      |
+| `<PrimaryButton text="Hello, world" />` | `<Button appearance='primary'>Hello, world</Button>` |
+| `<StealthButton text="Hello, world" />` | `<Button appearance='subtle'>Hello, world</Button>`  |
+
+### Props that remain as is
+
+- `children`
+- `disabled`
+- `icon`
+- `onClick`
```

Comment:
nit: I would vote to  rename this onPress, to better match pressable, but that's a followup for me on my other code review #1070 anyway

Reply:
Ah I forgot about that conversation. I think this will be a case of balancing alignment with web vs. what is intuitively native/RN, since I think web is named onClick to align with the HTML API.